include/exiv2/value.hpp

```diff
@@ -1250,7 +1250,7 @@ class ValueType : public Value {
     const auto v = value_.at(n);
     if (static_cast<decltype(v)>(std::numeric_limits<I>::min()) <= v &&
         v <= static_cast<decltype(v)>(std::numeric_limits<I>::max())) {
-      return static_cast<I>(std::round(v));
+      return static_cast<I>(std::lround(v));
```

Comment:
I deliberately didn't use `std::lround` here, because `long` is problematic, particularly on Windows. I considered using `std::llround` but decided against it because it's still not a platform-independent type. I wish the standard library had a platform-independent version of this that would convert to a type like `int64_t`.